integration-tests/each/__tests__/each-exception.test.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
you'll also need to execute them (and assert on the output). See e.g. https://github.com/facebook/jest/blob/master/integration-tests/__tests__/failures.test.js

Reply:
🤦‍♂️ I was wondering how they were ran 🤣

Reply:
Added in 6607413